assets/src/modules/map.js

```diff
@@ -479,6 +479,7 @@ export default class map extends olMap {
                                 })
                             });
                         } else {
+                            console.log(baseLayerState.name);
```

Comment:
Just a question, should we let this line ? i don't know what is the best practice.